pages/[user-slug]/[event-type-slug].tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react';
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import AccessTimeTwoToneIcon from '@mui/icons-material/AccessTimeTwoTone';
+import PublicTwoToneIcon from '@mui/icons-material/PublicTwoTone';
+import KeyboardArrowDownOutlinedIcon from '@mui/icons-material/KeyboardArrowDownOutlined';
+
+import DatePicker from '../../components/shared/DatePicker';
+
+const BookEvent = () => {
+  const router = useRouter();
+  const [value, setValue] = useState<Date | null>(new Date());
+  const [isSelectedValue, setIsSelectedValue] = useState(false);
```

Comment:
small thing and personal preference. I'd probably call it `isValueSelected`

Reply:
I will make sure that in the future I have proper variable renaiming